stablehlo/dialect/StablehloOps.td

```diff
@@ -2351,22 +2351,22 @@ def StableHLO_GatherOp: StableHLO_Op<"gather", [InferTensorTypeWithReify, Pure]>
 }
 
 def StableHLO_GetDimensionSizeOp: StableHLO_Op<"get_dimension_size",
-      [Pure, InferTensorType]> {
+      [Pure, InferTensorType /*get_dimension_size_c1*/]> {
```

Comment:
not sure if `c1` is appropriate in here? As it does not talk about the `result`. The constraint for `result` are all covered in the `Outputs` table entry and we do not have a label for the output like `i*`.